src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/messages.html.twig

```diff
@@ -27,7 +27,7 @@
 
 {% set messagesToShow = 4 %}
 
-<div class="card mt-2 d-print-none">
+<div id="messageCard" class="card mt-2 d-print-none">
```

Comment:
@boubkerbribri same, depends on what you decided, but we need to stick to the decision in the future

Reply:
We discuss it with @PierreRambaud, I think we will go with this pattern `js-test-*`, WDYT ?

Reply:
I like it, but I think @PierreRambaud didn't like it because it's adding some size to the loading and it's not respecting tests good practices because it's about modifying the code in order to test it

Reply:
I agree, editing the code for tests is not a good practice, more when you already have classes (`card`).
It can also break implementation as it adds an id on something that can be included more than once.

Reply:
And if you have no other choice, add `js-test` or something similar and avoid camelcase classes/ids :)